content/posts/visualising-usage-using-batteries/index.md

```diff
@@ -0,0 +1,220 @@
+---
+title: "Battery Charts - Visualise usage rates & more"
+date: 2021-08-19T16:52:58+05:30
+draft: false
+description: A tutorial on how to show usage rates and more using batteries
+categories: ["tutorials"]
+displayInList: true
+author: Rithwik Rajendran
+resources:
+-   name: featuredImage
+    src: "Liverpool_Usage_Chart.png"
+    params:
+        description: "my image description"
+        showOnTop: true
+
+---
+
+# Introduction
+
+I have been creating the cliche visualisations like scatter plots, bar charts, beeswarms etc. for a while and thought about doing something different. Since I'm an avid football fan, I thought of ideas to represent players' usage or involvement over a period (a season, a couple of seasons). I have seen some cool visualisations like donut which depict usage and I wanted to make something different and simple to understand. I thought about representing batteries as a form of player usage and it made a lot of sense.
+
+For players who have been barely used (less number of minutes played), show ***large amount of battery*** present since they have enough energy left in the tank. And for heavily used players, do the opposite i.e. show ***drained or less amount of battery***.
+
+So, what is the purpose of a battery chart? You can use it to show usage, consumption, involvement, fatigue etc. (anything usage related).
+
+The image below is a sample view of how a battery would look in our figure athough a single battery isn't exactly what we are going to recreate in this tutorial.
+
+![A sample visualisation](battery.png)
+
+# Tutorial
+
+Before jumping on to the tutorial, I would like to make it known that the function can be tweaked to fit accordingly depending on the number of subplots or any other size parameter. Coming to the figure we are going to plot, there are a series of steps that is to be considered which we will follow one by one. The following are those steps:-
+
+1. Outlining what we are going to plot
+2. Import necessary libraries
+3. Write a function to draw the battery
+    - This is the function that will be called to plot the battery chart
+4. Read the data and plot the chart accordingly
+    - We will demonstrate it with an example
+
+
+## <span style="text-decoration: underline">Plot Outline</span>
+
+What is our use case?
+
+- We are given a dataset where we have data of Liverpool's players and their minutes played in the last 2 seasons (for whichever club they played in that time period). We will use this data for our visualisation.
+- The final visualisation is the featured image of this blog post. We will navigate step-by-step to how we'll create the visualisation.
```

Comment:
Really like that you've started w/ the objective!

Reply:
Remove _The following are those steps:-_
Also, please remove - after : throughout